### PR TITLE
doc: add security release to banner

### PR DESCRIPTION
**Pull request by joesepi**
missed this in previous PR.

### CODE REVIEW

File: locale/en/site.json [hunk 1]
```diff
@@ -163,15 +163,10 @@
   },
   "banners": {
     "index": {
-      "startDate": "2022-01-04T16:00:00.000Z",
-      "endDate": "2022-01-25T16:00:00.000Z",
-      "text": "New security releases now available for Node.js 12, 14, 16, and 17 release lines",
-      "link": "https://nodejs.org/en/blog/vulnerability/jan-2022-security-releases/"
-    },
-    "blacklivesmatter": {
```
Comment (nschonni) [inline locale/en/site.json:171]: Just update the index banner, not this one

**Review by Trott — APPROVED** [review]

File: locale/en/site.json (hunk 1 above)
Comment (Trott) [reply locale/en/site.json:171]: This removes that banner info from site.json, which I think is OK, but if you want/need it in a separate PR, I guess we could do that too.

**Review by mhdawson — APPROVED** [review]
I agree with @Trott that it's likely ok to remove the other banner as well since it has not been active for a while.

**Comment by mhdawson** [comment]
@nschonni are you ok with the change as is? would be good to get the banner live for the security release pre-annouce.

**Comment by nschonni** [comment]
It's probably OK, but there was templating logic that still exists for that variable

File: locale/en/site.json [hunk 2]
```diff
@@ -163,15 +163,10 @@
...
-      "visible": false,
-      "text": "#BlackLivesMatter",
-      "link": "black-lives-matter/"
+      "startDate": "2022-03-14T12:00:00.000Z",
+      "endDate": "2022-03-28T12:00:00.000Z",
+      "text": "OpenSSL security releases may require Node.js security releases",
+      "link": "https://nodejs.org/en/blog/vulnerability/openssl-and-high-severity-fixes-mar-2022/"
     }
```
Comment (mhdawson) [inline locale/en/site.json]:
```suggestion
    }
    "blacklivesmatter": {
      "visible": false,
      "text": "#BlackLivesMatter",
      "link": "black-lives-matter/"
    }
```

File: locale/en/site.json (hunk 1 above)
Comment (mhdawson) [reply locale/en/site.json:171]: @nschonni just pushed a change to limit to the right part, look ok?

File: locale/en/site.json [hunk 3]
```diff
@@ -163,11 +163,11 @@
...
-      "startDate": "2022-01-04T16:00:00.000Z",
-      "endDate": "2022-01-25T16:00:00.000Z",
-      "text": "New security releases now available for Node.js 12, 14, 16, and 17 release lines",
-      "link": "https://nodejs.org/en/blog/vulnerability/jan-2022-security-releases/"
-    },
+      "startDate": "2022-03-14T12:00:00.000Z",
+      "endDate": "2022-03-28T12:00:00.000Z",
+      "text": "OpenSSL security releases may require Node.js security releases",
+      "link": "https://nodejs.org/en/blog/vulnerability/openssl-and-high-severity-fixes-mar-2022/"
+    }
```
Comment (mhdawson) [inline locale/en/site.json]:
```suggestion
    },
```

**Review by nschonni — APPROVED** [review]

**Comment by RDIL** [comment]
The link 404s.

**Pull request merged**